rts/Sim/Misc/ModInfo.cpp

```diff
@@ -43,6 +43,24 @@ void CModInfo::ResetState()
 		unitQuadPositionUpdateRate = 3;
 		groundUnitCollisionAvoidanceUpdateRate = 3;
 	}
+	{
+		// Default values for guard behavior (replicates the original behavior)
+		guardRecalculateThreshold = 100.0f;  // Distance that a guardee must move before the guard goal is recalculated
+		guardStoppedProximityGoal = 50.0f;  // Distance that a guardian will stop at nearing a stopped guardee
+		guardStoppedExtraDistance = 100.0f;  // The extra distance a guardian will keep from a stopped guardee
+		guardMovingProximityGoal = 150.0f;   // Distance the guardian is considered to be in guarding range and will match the velocity
+		guardMovingIntervalMultiplier = 0.0f;  // A multiplier for the moving goal while guarding, smaller values will result in higher detail movement but more performance cost
+		guardInterceptionLimit = 0.0f;        // Limit for the intercept when a guardian is not in guarding range
+	}
+	// {
+	// 	Recommended values for guard behavior (units keep formation and do not lag behind, slightly intercept)
+	// 	guardRecalculateThreshold = 100.0f;  // Distance that a guardee must move before the guard goal is recalculated
+	// 	guardStoppedProximityGoal = 50.0f;  // Distance that a guardian will stop at nearing a stopped guardee
+	// 	guardStoppedExtraDistance = 100.0f;  // The extra distance a guardian will keep from a stopped guardee
+	// 	guardMovingProximityGoal = 100.0f;   // Distance the guardian is considered to be in guarding range and will match the velocity
+	// 	guardMovingIntervalMultiplier = 2.13f;  // A multiplier for the moving goal while guarding, smaller values will result in higher detail movement but more performance cost
```

Comment:
note i've updated this value due to the incorporation of the GAME_SPEED later on